adg/pipelines/scripts/edx-run-test.sh

```diff
@@ -28,24 +28,76 @@ branch="$1"
 # There is no need to install the prereqs, as this was already
 # just done via the dependencies override section of circle.yml.
 export NO_PREREQ_INSTALL='true'
+export PARALLEL_RUNS_COMPLETED=0
+EXIT=0
 
-# run Edx common lib unit tests
-paver test_lib --cov-args="-p" --skip-clean --cov-config=.coveragerc-local --cov-append
+if [ "$CIRCLE_NODE_TOTAL" == "1" ] ; then
+    echo "Only 1 container is being used to run the tests."
+    echo "To run in more containers, configure parallelism for this repo's settings "
+    echo "via the CircleCI UI and adjust scripts/circle-ci-tests.sh to match."
```

Comment:
Let's not echo this as it is an internal detail. Instead let's leave this here as a comment